### PR TITLE
rm!(install-local): merge `-Il` functionality with `-I`

### PR DESCRIPTION
**Pull request by Henryws**
## Purpose

Merge them into one flag.

## Checklist

- [x] I confirm that I have read the [contributing guidelines](https://github.com/pacstall/pacstall/blob/develop/CONTRIBUTING.md), and this pull request is abiding by all the clauses stated in the guideline.

### CODE REVIEW

File: pacstall [hunk 1]
```diff
@@ -492,7 +490,38 @@ Helpful links:
...
+                    fi
+                    if [[ -s $LOGFILE ]]; then
+                        exit 1
+                    fi
+                    continue
+                else
+                    export type="install"
+                    export local="no"
+                    export PACKAGE=$2
+                fi
```
Comment (D-Brox) [inline pacstall:523]: Lines 525-559 should all be inside this else

File: pacstall [hunk 2]
```diff
@@ -492,7 +490,38 @@ Helpful links:
...
+                        continue
+                    fi
+                    if ! source "$STGDIR/scripts/install-local.sh"; then
+                        fancy_message error "Failed to install ${GREEN}${PACKAGE}${NC}"
+                        exit 1
+                    fi
+                    if [[ -s $LOGFILE ]]; then
+                        exit 1
+                    fi
+                    continue
```
Comment (D-Brox) [inline pacstall:518]: This is could be outside the if, since it's used by both local and remote installs

**Review by D-Brox — CHANGES_REQUESTED** [review]

File: pacstall [hunk 3]
```diff
@@ -492,97 +490,82 @@ Helpful links:
...
+                        export fail_install
+                        fancy_message error "Failed to download the ${GREEN}${PACKAGE}${NC} pacscript"
+                        continue
+                    fi
+
+                    export REPO
+                    if ! source "$STGDIR/scripts/install-local.sh"; then
+                        fancy_message error "Failed to install ${GREEN}${PACKAGE}${NC}"
+                        exit 1
+                    fi
```
Comment (D-Brox) [inline pacstall:560]: This is outside the else

File: pacstall [hunk 4]
```diff
@@ -492,97 +490,82 @@ Helpful links:
...
+                        shift
+                        continue
+                    fi
+                    if ! source "$STGDIR/scripts/install-local.sh"; then
+                        fancy_message error "Failed to install ${GREEN}${PACKAGE}${NC}"
+                        exit 1
+                    fi
+                    if [[ -s $LOGFILE ]]; then
+                        exit 1
+                    fi
```
Comment (D-Brox) [inline pacstall:517]: This is outside the if

**Review by D-Brox — CHANGES_REQUESTED** [review]

**Review by D-Brox — APPROVED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**